arwen/context/arwen.go

```diff
@@ -708,6 +738,13 @@ func (host *vmContext) WriteLog(addr []byte, topics [][]byte, data []byte) {
 	host.logs[strAdr] = currLogs
 }
 
+func (host *vmContext) SetAsyncCallInfo(dest []byte, value []byte, gasLimit uint64, data []byte) {
```

Comment:
this could have its own component.

Reply:
Created the struct `AsyncCallInfo`, part of the `runtimeContext`.